OSDOCS-17178: SEC-10: CQA 2.0 - Security Profile Operator (#112027)
openshift/openshift-docs: security/security_profiles_operator/spo-advanced.adoc

toc::[]

[role="_abstract"]

> This section explains how to use advanced Security Profiles Operator tasks to enable metrics, configure webhooks, or restrict syscalls.

Reviewer: same thing about self referential short descriptions:
```suggestion
You can use advanced Security Profiles Operator tasks to enable metrics, configure webhooks, or restrict syscalls.
```

Author: Done, thank you.
